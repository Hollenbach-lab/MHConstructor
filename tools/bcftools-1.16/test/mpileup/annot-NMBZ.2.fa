>chr6 157203791-157203940
TACTCTTTCGTTAACTTTCGTTCTTTCATGCATAGAGTCAACATTCATGATATCCTTGTT
CTTCCCCATCTTCAGTTACTCCTGAGGCGTGGCGTGTGATGATGTCCCTTAAATCAGGTC
TTTTGGCTGAGAGTACGTGGGCTTTGGACA
