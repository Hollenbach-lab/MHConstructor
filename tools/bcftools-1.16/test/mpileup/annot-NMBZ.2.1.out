##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=chr6,length=150>
##ALT=<ID=*,Description="Represents allele(s) other than observed.">
##INFO=<ID=INDEL,Number=0,Type=Flag,Description="Indicates that the variant is an INDEL.">
##INFO=<ID=IDV,Number=1,Type=Integer,Description="Maximum number of raw reads supporting an indel">
##INFO=<ID=IMF,Number=1,Type=Float,Description="Maximum fraction of raw reads supporting an indel">
##INFO=<ID=DP,Number=1,Type=Integer,Description="Raw read depth">
##INFO=<ID=VDB,Number=1,Type=Float,Description="Variant Distance Bias for filtering splice-site artefacts in RNA-seq data (bigger is better)",Version="3">
##INFO=<ID=RPBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Read Position Bias (closer to 0 is better)">
##INFO=<ID=MQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality Bias (closer to 0 is better)">
##INFO=<ID=BQBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Base Quality Bias (closer to 0 is better)">
##INFO=<ID=MQSBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Mapping Quality vs Strand Bias (closer to 0 is better)">
##INFO=<ID=NMBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Number of Mismatches within supporting reads (closer to 0 is better)">
##INFO=<ID=SCBZ,Number=1,Type=Float,Description="Mann-Whitney U-z test of Soft-Clip Length Bias (closer to 0 is better)">
##INFO=<ID=FS,Number=1,Type=Float,Description="Phred-scaled p-value using Fisher's exact test to detect strand bias">
##INFO=<ID=SGB,Number=1,Type=Float,Description="Segregation based metric.">
##INFO=<ID=MQ0F,Number=1,Type=Float,Description="Fraction of MQ0 reads (smaller is better)">
##INFO=<ID=I16,Number=16,Type=Float,Description="Auxiliary tag used for calling, see description of bcf_callret1_t in bam2bcf.h">
##INFO=<ID=QS,Number=R,Type=Float,Description="Auxiliary tag used for calling">
##FORMAT=<ID=PL,Number=G,Type=Integer,Description="List of Phred-scaled genotype likelihoods">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
chr6	75	.	G	A,<*>	0	.	DP=283;I16=131,25,98,28,5557,200289,4591,168791,9360,561600,7560,453600,2484,51052,2121,45743;QS=0.547596,0.452404,0;VDB=1.58389e-09;SGB=-0.693147;RPBZ=-0.269598;MQBZ=0;MQSBZ=0;BQBZ=1.5258;NMBZ=-2.02395;SCBZ=-1.56217;FS=0;MQ0F=0	PL	255,0,255,255,255,255
